src/TrafficSign/TrafficSign.tsx

```diff
@@ -0,0 +1,38 @@
+import Figure from "../Figure";
+import React from "react";
+import { TrafficSignProps } from "./TrafficSign.types";
+import { getRoadSignName } from "./TrafficSignHelper";
+
+/**
+ * A UI component that displays an SVG of the road sign on a canvas.
+ * @param angle The angle of rotation of the image in degrees
+ * @param type The name of the road sign
+ * @param value The value within the road sign (e.g., "50") for a speed limit
+ * @param points List of coordinates of the image on the canvas [x,y]
+ * @param scale A number to scale the image, keeping the aspect ratio
+ * @param size An object that allows sizing x and y independently
+ * @returns React Konva Image component
```

Comment:
No need to document all of props here when the input is an argument. We document for positional arguments but for something that takes an object just document the object type (TrafficSignProps) and then only document each prop in the type definition itself. Just duplicating effort otherwise.

Reply:
Should these be the defaults of the component? For example the default value for `scale` of trafficSign is `undefined`. This is so because the underlying component `Figure` will default `scale` to 3. So should the default value for `scale` for `TrafficSign` be `undefined` or `3` for storybook?

Reply:
No defaults set to `undefined`